sysdig/internal/client/v2/model.go

```diff
@@ -411,6 +415,9 @@ type CloudAccountSecure struct {
 	WorkLoadIdentityAccountID    string `json:"workloadIdentityAccountId,omitempty"`
 	WorkLoadIdentityAccountAlias string `json:"workLoadIdentityAccountAlias,omitempty"`
 }
+
+type CloudauthAccountSecure cloudauth.CloudAccount
```

Comment:
Let's not have Cloudauth in the name here, can you rename it to something generic?

Reply:
Actually, this PR has both my and Raghav's changes. I had renamed files and references to cloudauth in `client` changes from this suggested comment - https://github.com/sysdiglabs/terraform-provider-sysdig/pull/404#pullrequestreview-1613952955